Rakefile

```diff
@@ -52,6 +54,10 @@ class Component
     @path.join('app.js').exist?
   end
 
+  def rust_app?
```

Comment:
There needs to be something added to `build` namespace in this main Rakefile that will also build the ood-proxy-rs app.  This way the existing `rake build` inside the RPM spec will pick up this new app.  

One idea I can imagine is changing the `infrastructure` function from array generating components to hash where key is app name like in the array and value is the name of the build task, then pass that extra info into Component and reference the build task name somewhere in build namespace after looping over infrastructure apps.